source/utils/timed-out.ts

```diff
@@ -31,7 +31,7 @@ export interface Delays {
 export default (request: ClientRequest, delays: Delays, options: any) => {
 	/* istanbul ignore next: this makes sure timed-out isn't called twice */
 	if (Reflect.has(request, reentry)) {
-		return;
+		return noop;
```

Comment:
Returning a no-op function certainly is safer than returning something that is not callable! 😀 

It might be even safer to find a way to return a second cancellation handle for the same timeout. I'm not sure it's worth it, or for that matter really clear on the circumstances that could lead here.

Reply:
Returning a no-op function is fine.